lte/gateway/c/core/oai/common/common_ies.h

```diff
@@ -42,6 +42,8 @@
 #define MAX_VLR_NAME_LENGTH 255
 #define SGS_ASSOC_ACTIVE 1
 #define SGS_ASSOC_INACTIVE 0
+#define MAX_IMEISV_SIZE 16
+#define MAX_IMEI_SIZE 15
```

Comment:
These are already defined on lines 39 and 40.

Reply:
comment addressed

Reply:
I do not see that it is addressed. Did you submit your changes?

Reply:
code is not yet submitted, was doing sanity. I will submit